svc-events/events/publishevttosubscriber.go

```diff
@@ -187,20 +194,19 @@ func filterEventsToBeForwarded(subscription evmodel.Subscription, events []byte,
 
 // formatEvent will format the event string according to the odimra
 // add uuid:systemid/chassisid inplace of systemid/chassisid
-func formatEvent(originResource, eventstring, hostIP string) (string, string) {
+func formatEvent(originResource string, event common.Event, hostIP string) (common.Event, string) {
```

Comment:
along with OriginOfCondtion, the resource uri might be present in OEM block of the  event  that use case is  not handled